sunpy/map/header_helper.py

```diff
@@ -384,8 +384,9 @@ def _set_instrument_meta(meta_wcs, instrument, telescope, observatory, detector,
     'Matrix element CDi_j describing the rotation required to align solar North with the top of the image.'
 }
 
-
-def make_heliographic_header(date, observer_coordinate, shape, *, frame, projection_code="CAR"):
+@u.quantity_input
+def make_heliographic_header(date, observer_coordinate, shape, *, frame, projection_code="CAR",
+                             map_center_longitude: u.Quantity['angle'] = 0.0*u.deg):
```

Comment:
```suggestion
                             map_center_longitude: u.Quantity[u.deg] = 0.0*u.deg):
```
Specifying a string makes `ruff` unhappy, so let's just specify a specific angle unit instead.  It doesn't affect the behavior at all, just the error message.